### ISSUE TITLE
Clicking on someones name shows where they are

### ISSUE THREAD

**Issue by GrantBartlett** [CnCNet/xna-cncnet-client#51]
It was quite a nice little feature back in the very original client, seeing where other players are.  If they're in a game or in the lobby etc.

It would be quite nice to see this in the core client again.

**Comment by Rampastring** [comment]
Good idea. Although rather than clicking on someone's name, I'd prefer if it was hover-on info (like the game information) or something like that. But I'll implement it in some way once I can.

**Comment by Martin-Veskilt** [comment]
Some players have been requesting this feature in the YR community.

**Comment by Metadorius** [comment]
Thanks for heads up, we're aware of the requests.

**Comment by devo1929** [comment]
FYI
https://github.com/CnCNet/xna-cncnet-client/pull/228

That PR provides the ability to join another user. It doesn't accomplish the specific request here, but it already has the necessary code to "find" the game that the selected user is in. Thus, allowing you to quickly add this feature as well.
